data/data.go

```diff
@@ -4,13 +4,25 @@ type Board [5][5]Holder
 
 // Returns a new 2D board
 func NewBoard() *Board {
-	return nil
+	board := Board{}
+	return &board
```

Comment:
Can look into the `New` keyword or even directly return the pointer

Reply:
Nah no need these are just suggestions you could have used instead

Reply:
Okay! Will definitely keep them in mind while working on future projects! :)